CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # CHANGELOG
 
 # Version 2.2.0
-- Introduces SpringBoot Starter 1.0.2-BETA
+- Introduces SpringBoot Starter 1.1.0-BETA
 - Starter now respects autoconfiguration for Micrometer metrics.
+- Starter adds autoconfiguration for Local Forwarder gRPC Telemetry Channel. (Please look at readme for details on configuration.)
```

Comment:
We shouldn't need to qualify it with gRPC.

Reply:
done